src/bepasty/cli/server.py

```diff
@@ -16,6 +17,9 @@ def main():
 
     args = argparser.parse_args()
     app = create_app()
+
+    mimetypes.add_type('application/x-asciinema-recording', '.cast')
```

Comment:
better, but still not the right place.

i guess this gets only invoked when starting the built-in server from flask, but not when using this with an external server (gunicorn, uswsgi, mod-wsgi) just as a wsgi app.

Reply:
did you see my comment?

Reply:
check if this should be on top level of module, where mimetypes is first imported.
maybe right below the import statement.

Reply:
fixed this myself.